### PR TITLE
[TT-1021] Add new Tyk Hybrid Gateway config info

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/tyk-cloud/environments-&-deployments/hybrid-gateways.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Introduction
+
+Hybrid Gateways are available on our [14 Day Free Trial](/docs/tyk-cloud/account-billing/plans/#14-day-trial) and [Enterprise Global](/docs/tyk-cloud/account-billing/plans/#enterprise-global-plan) plans. There are some settings you need to be aware of when adding a Hybrid Gateway to a Tyk Cloud installation:
+
+## Configuration
+
+* The MDCB endpoint URL is unique for each control plane
+* The TLS certificate is also unique and currently self-signed (i.e. not globally trusted), so you need to place it in the host trust store or by setting the Tyk Gateway config option `slave_options.ssl_insecure_skip_verify` to `true`.
+* The Hybrid Gateway should not bind to slugs. In the Hybrid Gateway config, set `slave_options.bind_to_slugs` to `false`.
+* Any [monitor](/docs/tyk-oss-gateway/configuration/#monitor) configuration settings with calling back to the old Tyk Cloud should be removed.
```
Comment (joshblakeley) [inline tyk-docs/content/tyk-cloud/environments-&-deployments/hybrid-gateways.md]: We can remove this I think - was more of an internal notes from artem

File: tyk-docs/content/tyk-cloud/environments-&-deployments/hybrid-gateways.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        "port": 6379,
+        "enable_cluster": false,
+        "username": "",
+        "password": "",
+        "database": 0,
+        "optimisation_max_idle": 1000
+    },
+    "enable_analytics": true,
+    "analytics_config": {
+        "type": "rpc",
```
Comment (joshblakeley) [inline tyk-docs/content/tyk-cloud/environments-&-deployments/hybrid-gateways.md:38]: Half the config seems to have been chopped off

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/setup-slave-data-centres.md [hunk 3]
```diff
@@ -77,7 +77,7 @@ The most important elements here are:
...
 |`group_id`    |This is the "zone" that this instance inhabits, e.g. the cluster/data-centre the gateway lives in. The group ID must be the same across all the gateways of a data-centre/cluster which are also sharing the same Redis instance. This id should also be unique per cluster (otherwise another gateways cluster can pick up your keyspace events and your cluster will get zero updates).
 |`connection_string`     |The MDCB instance or load balancer.|
-| `bind_to_slugs` | For on-premise installation it is expected to be `false`. For Multi-cloud gateways (using Tyk's control plane) it MUST be `true` |
+| `bind_to_slugs` | For Tyk On-Premises this should be t to `false`. For our previous Multi-Cloud Gateways (using Tyk's control plane) it MUST be `true`. For our new Tyk Cloud Hybrid Gateway setup, see [here](/docs/tyk-cloud/environments-deployments/hybrid-gateways/#configuration). |
```
Comment (joshblakeley) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/setup-slave-data-centres.md]: Refactor to something like 

For all Tyk installations except for Classic Cloud this should be set to  `false`.

**Review by joshblakeley — COMMENTED** [review]
Thanks @marksou - just a few things to catch and were golden

File: tyk-docs/content/tyk-cloud/environments-&-deployments/hybrid-gateways.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/tyk-cloud/environments-&-deployments/hybrid-gateways.md]: Done

File: tyk-docs/content/tyk-cloud/environments-&-deployments/hybrid-gateways.md (hunk 2 above)
Comment (marksou) [reply tyk-docs/content/tyk-cloud/environments-&-deployments/hybrid-gateways.md:38]: Fixed

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/setup-slave-data-centres.md (hunk 3 above)
Comment (marksou) [reply tyk-docs/content/tyk-stack/tyk-multi-data-centre/setup-slave-data-centres.md]: Done

**Review by joshblakeley — APPROVED** [review]
Good as a start thanks @marksou.
 
SA team can iterate during the week

**Pull request merged**